src/Sprite.js

```diff
@@ -305,6 +304,10 @@ class SpriteBase {
   get answer() {
     return this._project.answer;
   }
+
+  async loudness() {
+    return this._project.loudnessHandler.getLoudness();
```

Comment:
It looks like Scratch [only recomputes loudness once per step](https://github.com/LLK/scratch-vm/blob/f405e59d01a8f9c0e3e986fb5276667a8a3c7d40/src/blocks/scratch3_sensing.js#L262-L275).